airflow/hooks/hive_hooks.py

```diff
@@ -121,11 +148,31 @@ def run_cli(self, hql, schema=None, verbose=True, hive_conf=None):
                 for key, value in hive_conf.items():
                     cmd_extra += ['-hiveconf', '{0}={1}'.format(key, value)]
 
-                hive_cmd = [hive_bin, '-f', fname] + cmd_extra
+                hive_cmd = [hive_bin, '-f', fname]
 
                 if self.hive_cli_params:
                     hive_params_list = self.hive_cli_params.split()
-                    hive_cmd.extend(hive_params_list)
+                else:
+                    hive_params_list = []
+
+                if self.mapred_queue:
+                    hive_params_list.append(
```

Comment:
If you move these lines immediately above the for loop that adds the ```hive_conf``` parameters to the ```cmd_extra``` list (between 147 and 148) you could shorten the body of the ifs to ```hive_conf['mapreduce.job.queuename'] = self.mapred_queue``` etc. Just a small style change but IMHO it would be a little cleaner.